dev-python/shiboken/shiboken-5.11.1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit cmake-utils llvm python-r1
+
+DESCRIPTION="Tool for creating Python bindings for C++ libraries"
+HOMEPAGE="https://wiki.qt.io/PySide2"
+SRC_URI="https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${PV}-src/pyside-setup-everywhere-src-${PV}.tar.xz"
+
+# The "sources/shiboken2/libshiboken" directory is triple-licensed under the GPL
+# v2, v3+, and LGPL v3. All remaining files are licensed under the GPL v3 with
+# version 1.0 of a Qt-specific exception enabling shiboken2 output to be
+# arbitrarily relicensed. (TODO)
+LICENSE="|| ( GPL-2 GPL-3+ LGPL-3 ) GPL-3"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
+IUSE="numpy test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}
+	numpy? ( dev-python/numpy[${PYTHON_USEDEP}] )
```

Comment:
we usually sort conditional deps after unconditional ones, IOW move this line last